### ISSUE TITLE
AVAX sent to wrong chain, recoverable?

### ISSUE THREAD

**Issue by mrgreedy911** [ava-labs/avalanche-wallet#171]
I got the coinbase shit and they never state which network they send on so i sent AVAX on the wrong network
https://snowtrace.io/tx/0xee3626b2d4c9d80d9effaf555c348c92c069d743780aa38dc9058ad1ffef16b5
Coinbase sent to a bep20 address over the AVAX network...
Can these funds be recovered?

**Comment by hanahdev** [comment]
> I got the coinbase shit and they never state which network they send on so i sent AVAX on the wrong network https://snowtrace.io/tx/0xee3626b2d4c9d80d9effaf555c348c92c069d743780aa38dc9058ad1ffef16b5 Coinbase sent to a bep20 address over the AVAX network... Can these funds be recovered?

yes it can be recovered kindly text there support on whatsapp - +19546679281
